src/dlgprefcontrols.cpp

```diff
@@ -89,25 +89,35 @@ DlgPrefControls::DlgPrefControls(QWidget * parent, MixxxMainWindow * mixxx,
     connect(ComboBoxRateDir, SIGNAL(activated(int)),
             this, SLOT(slotSetRateDir(int)));
 
-    // Set default range as stored in config file
-    if (m_pConfig->getValueString(ConfigKey("[Controls]", "RateRange")).length() == 0)
-        m_pConfig->set(ConfigKey("[Controls]", "RateRange"),ConfigValue(2));
-
     ComboBoxRateRange->clear();
-    ComboBoxRateRange->addItem(tr("6%"));
-    ComboBoxRateRange->addItem(tr("8% (Technics SL-1210)"));
-    ComboBoxRateRange->addItem(tr("10%"));
-    ComboBoxRateRange->addItem(tr("20%"));
-    ComboBoxRateRange->addItem(tr("30%"));
-    ComboBoxRateRange->addItem(tr("40%"));
-    ComboBoxRateRange->addItem(tr("50%"));
-    ComboBoxRateRange->addItem(tr("60%"));
-    ComboBoxRateRange->addItem(tr("70%"));
-    ComboBoxRateRange->addItem(tr("80%"));
-    ComboBoxRateRange->addItem(tr("90%"));
+    ComboBoxRateRange->addItem(tr("4% (Denon DN-D4500)"), 4);
+    ComboBoxRateRange->addItem(tr("6% (Pioneer CDJ/DDJ-S1)"), 6);
+    ComboBoxRateRange->addItem(tr("8% (Technics SL-1210, Pioneer DDJ-SX)"), 8);
```

Comment:
Taking the SL-1210 as reference here is a good choice because it is THE standard DJ turntable. 
The other references feel somehow random, because they have a configurable pitch scale, I would just delete them.
6% could be "(Halftone)"